crypto/src/main/kotlin/web5/sdk/crypto/KeyManager.kt

```diff
@@ -52,7 +53,7 @@ public interface KeyManager {
    * practices and safeguarding the private key during the operation. The specific signing algorithm
    * used may depend on the type and parameters of the stored key.
    */
-  public fun sign(keyAlias: String, payload: Payload)
+  public fun sign(keyAlias: String, payload: Payload): JWSObject
```

Comment:
I think you may have some merge conflicts with the pr because I see:

https://github.com/TBD54566975/web5-kt/pull/63/files#diff-a76ca21a2d268f0c593a56f6a40fa7ab54cabe5405bbc08a37e14ffee6e74006R93

    val signatureBytes = did.keyManager.sign(keyAlias, toSign)

Reply:
Yes, they've been fixed now.